src/main/java/mate/academy/onlinebookstore/entity/CartItem.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.onlinebookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Data
```

Comment:
dont use Data annotation on entities, as it cause bugs. replace it with getter/setter annotations instead